Makefile

```diff
@@ -133,7 +133,7 @@ else
 	mkdir -p ./bin/$(TARGETOS)_$(TARGETARCH)
 	$(CXX) -o ./bin/$(TARGETOS)_$(TARGETARCH)/syz-executor$(EXE) executor/executor.cc \
 		$(ADDCXXFLAGS) $(CXXFLAGS) $(LDFLAGS) -DGOOS_$(TARGETOS)=1 -DGOARCH_$(TARGETARCH)=1 \
-		-DHOSTGOOS_$(HOSTOS)=1 -DGIT_REVISION=\"$(REV)\"
+		-DHOSTGOOS_$(HOSTOS)=1 -DGIT_REVISION=\"$(REV)\" -fno-jump-tables
```

Comment:
We did not do this b/c we don't distribute required compilation flags with reproducers. Instead we tried to prevent compiler from emitting jump tables on the source code level:
```
// Prevent function inlining. This attribute is applied to every guest_handle_* function,
// making sure they remain small so that the compiler does not attempt to be too clever
// (e.g. generate switch tables).
#define noinline __attribute__((noinline))
```

Where does it emit them now?